dockerfiles/gomoderator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM golang:alpine as builder
+
+RUN apk --update --no-cache add make git g++ linux-headers python3
+
+ARG VDB_VERSION=staging
+ENV GO111MODULE on
+
+WORKDIR /go/src/github.com/makerdao
+RUN git clone https://github.com/makerdao/vulcanizedb.git
+RUN git clone https://github.com/makerdao/vdb-mcd-transformers.git
+WORKDIR /go/src/github.com/makerdao/vulcanizedb
+RUN git checkout $VDB_VERSION
+RUN go build
+WORKDIR /go/src/github.com/makerdao/vdb-mcd-transformers
+RUN git checkout $VDB_VERSION
+RUN go build
+
+# app container
+FROM golang:alpine
+WORKDIR /go/src/github.com/makerdao/vulcanizedb
+
+# add certificates for node requests via https
+# bash for wait-for-it
+RUN apk update \
+        && apk upgrade \
+        && apk add --no-cache \
+        ca-certificates \
+        bash \
+        && update-ca-certificates 2>/dev/null || true
+
+# add go so we can build the plugin
+RUN apk add --update --no-cache git g++ linux-headers python3
+
+# get access to go.mod files in directories
+COPY --from=builder /go/src/github.com/makerdao/vulcanizedb .
+COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/go.mod ./vdb/go.mod
```

Comment:
Is `./vdb/go.mod` making it so that you're putting the vdb-mcd-transformers' go.mod in the same dir, so it's easier to call the gomoderator below?

Super nitpicky, but I wonder if we could change `./vdb/go.mod` to `./mcd-transformers/go.mod`? Don't feel super strongly about it, but it just confused me at first that this is the transformers' go.mod file, not the vulcanizedb one.

Reply:
yes on `./vdb/go.mod`. Is that also confusing? I couldn't figure out how to get the PATH working correctly but could try again if this is a confusing step.

Not nitpicky at all on the dir naming. That's why we have another set of 👀

Reply:
Copying the vdb-mcd-transformers go.mod over to the current directory makes sense to me!